### ISSUE TITLE
"Nuke option" - delete local repositories before cloning them from upstream.

### ISSUE THREAD

**Issue by kavink** [akhikhl/multiproject-git-gradle#15]
It will be good to have a "Nuke option" to delete and recline repositories. i.e. multiproject-git-gradle should have its own local folder called something like "localSandbox" , It will clone repos in this folder do releases etc. And nuke option is called, it will delete repos from this folder and reclone , checkout required release branch and do releases

Please add this folder in .gitignore folder on multiproject-git-gradle, so its never committed in.

**Comment by akhikhl** [comment]
Resolved in version 1.0.20.

Now multiproject-git-gradle supports new task: "nuke". It can be combined with other tasks like this:

```
gradle nuke release
```

in the example above we remove all local git repositories listed in "config.gradle", then proceed with normal release process.

**Comment by kavink** [comment]
Thanks! which folder does this work on ?

**Comment by akhikhl** [comment]
It works on the set of local git repositories listed in "config.gradle".
Optional property "rootDir" defines where local repositories are located
(the default is '.' - that means: local git repositories are located in the
same folder as multiproject-git-gradle).

**Comment by kavink** [comment]
:+1:

**Comment by kavink** [comment]
I just am trying to test this. I don think nuke is deleting the folders. Or it might be a bug in  Issue #13 As i don't seem to have latest data in my local repo

**Comment by akhikhl** [comment]
Fixed in version 1.0.21. I tested it in these combinations:

```
gradle nuke
gradle nuke build
gradle nuke update build
gradle nuke release
```

**Comment by kavink** [comment]
This is working as expected. Normal workflow without nuke fails for me. Ill create another issue for it.
